api/src/pcapi/core/mails/transactional/bookings/booking_event_reminder_to_beneficiary.py

```diff
@@ -1,6 +1,6 @@
 from pcapi.core import mails
-import pcapi.core.bookings.api as bookings_api
 from pcapi.core.bookings.models import Booking
+import pcapi.core.bookings.utils as bookings_utils
```

Comment:
Détail : pour que ce soit bien homogène : 
```suggestion
from pcapi.core.bookings import utils as bookings_utils
```